cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -142,7 +142,8 @@ exporters:
   # Send to gateway
   otlp:
     endpoint: "${SPLUNK_GATEWAY_URL}:4317"
-    insecure: true
+    tls:
+      insecure: true
```

Comment:
I believe this is something we need to worry about. 

If user upgrades linux package collector likely won't be able to start. Is this desired behavior?

If not we can probably add another case in`internal/configconverter` to handle that?

cc @emaderer @flands  @jcheng-splunk @rmfitzpatrick

Reply:
Just confirmed: collector fails to start with the old config even if the `otlp` exported is not added to the pipeline by default

Reply:
Hi @dmitryax, I just pushed a change to move that config key.

Reply:
Thanks @pmcollins !